primer/brand
# Update devcontainer image version `typescript-node:20`

## Summary

Node 18 needed for Storybook 8 to be run. Current node version is 16, therefore the image version for the container is upped to 18. I just wanted to be able to run `npm run start` right away after Codespaces loads.

## List of notable changes:



- **updated** dev container image version from `typescript-node:16` to `typescript-node:20`

## What should reviewers focus on?



- Verify that the `node -v` is indeed 20

## Steps to test:



1. Create a new codespace
1. Run `node -v`
1. Should say `v20.x.x`

P.S. Tested it myself works.

## Supporting resources (related issues, external links, etc):

-
-

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">



 </td>
<td valign="top">



</td>
</tr>
</table>
